src/suites/cts/copyImageBitmapToTexture.spec.ts

```diff
@@ -0,0 +1,168 @@
+export const description = `
+copy imageBitmap To texture tests.
+`;
+
+import { TestGroup } from '../../framework/index.js';
+
+import { GPUTest } from './gpu_test.js';
+
+class F extends GPUTest {
+  calculateRowPitch(width: number, bytesPerPixel: number): number {
```

Comment:
nit: can be a standalone function or static method (does not use `this`).

Reply:
Done.